src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1470,6 +1470,16 @@ void PolicyHandler::SetRemoteControl(bool enabled) {
   POLICY_LIB_CHECK_VOID();
   policy_manager_->SetRemoteControl(enabled);
 
+  OnCountryConsentChanged(enabled);
```

Comment:
The method OnCountryConsentChanged contains only code from SetRemoteControl after line "policy_manager_->SetRemoteControl" - nothing changed from SetRemoteControl's perspective.